lib/lightning_web/live/components/layout.ex

```diff
@@ -23,8 +23,9 @@ defmodule LightningWeb.Components.Layout do
     ~H"""
     <div class="flex-none bg-white shadow-sm z-20">
       <div class="max-w-7xl mx-auto h-20 sm:px-6 lg:px-8 flex items-center">
-        <h1 class="text-3xl font-bold text-secondary-900">
+        <h1 class="text-3xl font-bold text-secondary-900 flex items-center">
           <%= @title %>
+          <%= if assigns[:details], do: render_slot(@details) %>
```

Comment:
I think we should avoid adding to the component interface. (`:details` slot vs `title` attr).

I prefer the named slot approach you just added, could you change the slot name to `:title` and then change all the templates to use `<:title>...</:title>` please.

Reply:
Agreed. Had some hesitations of changing all the page titles.